libparsec/crates/protocol/tests/authenticated_cmds/v3/message_get.rs

```diff
@@ -1,15 +1,74 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+use libparsec_tests_lite::hex;
+
 use super::authenticated_cmds;
 
 // Request
 
 pub fn req() {
-    // TODO #4545: Implement test
+    // Generated from Python implementation (Parsec v2.6.0+dev)
+    // Content:
+    //   cmd: "message_get"
+    //   offset: 8
+    let raw = hex!("82a3636d64ab6d6573736167655f676574a66f666673657408");
+
+    let req = authenticated_cmds::message_get::Req { offset: 8 };
+
+    let expected = authenticated_cmds::AnyCmdReq::MessageGet(req);
+
+    let data = authenticated_cmds::AnyCmdReq::load(&raw).unwrap();
+
+    assert_eq!(data, expected);
+
+    let authenticated_cmds::AnyCmdReq::MessageGet(req2) = data else {
+        unreachable!()
+    };
+
+    let raw2 = req2.dump().unwrap();
+
+    let data2 = authenticated_cmds::AnyCmdReq::load(&raw2).unwrap();
+
+    assert_eq!(data2, expected);
```

Comment:
use `p_assert_eq!` instead from `libparsec_test_lite`

Reply:
@touilleMan will tell you to use: ```Rust
use libparsec_tests_lite::prelude::*
```

Reply:
done